### PR TITLE
added media query for the first row

### PR DESCRIPTION
**Pull request by chloezzy**
我正在写第一个row，下面是我遇到的几个问题：

1. 两边儿的spacing怎么写，margin和padding各写多少？
2. 怎么让图片在container外面的部分不见，我查到了可以用overflow: hidden，但是我不确定怎么写。是写img在html，还是写一个container，css里写background- image呢？

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,18 +34,22 @@
     <main>
         <div class="container">
             <div class="row first">
-                <div class="col-6">
+                <div class="col-6 d-flex flex-column justify-content-center" id="textFirst">
                     <h1>TYPEMASTER KEYBOARD</h1>
                     <p>Improve your productivity and gaming without breaking the bank. Upgrade to a high quality
                         mechanical typing experience.</p>
-                    <div class="container">
+                    <div class="container" id="buttons">
```
Comment (DxSnow) [inline index.html:41]: line42: what is text-end?

**Review by DxSnow — COMMENTED** [review]
questions about line33(why left padding?)and line42 (why text-end?)

File: index.html (hunk 1 above)
Comment (chloezzy) [reply index.html:41]: text-end aligns the text to the end of its container, in this case, the text in the button align to the right of the button

File: app.css [hunk 2]
```diff
@@ -52,3 +52,22 @@ h3 {
     flex-direction: column;
```
[the remark this answers is not in the document]
Comment (chloezzy) [reply app.css:52]: ok sure, I will leave it without the padding.

**Pull request merged**